dotnet/src/SemanticKernel.Core/Functions/NativeFunction.cs

```diff
@@ -138,7 +139,21 @@ public FunctionView Describe()
     {
         try
         {
-            return await this._function(null, requestSettings, context, cancellationToken).ConfigureAwait(false);
+            this.CallFunctionInvoking(context);
+            if (SKFunction.IsInvokingCancelOrSkipRequested(context))
+            {
+                return new FunctionResult(this.Name, this.PluginName, context);
+            }
+
+            var invokeResult = await this._function(null, requestSettings, context, cancellationToken).ConfigureAwait(false);
+            var finalResult = this.CallFunctionInvoked(invokeResult, context);
+
```

Comment:
Move empty line to 149, males more sense to keep the `CallFunctionInvoked` beside the `if` check